recipe/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set python = "python >=3.9" %}
+
+package:
+  name: uwtools
+  version: 0.1.0
+source:
+  path: ../src
+build:
+  include_recipe: false
+  noarch: python
+  number: 0
+requirements:
+  host:
+    - pip
+    - {{ python }}
+  run:
+    - boto3 1.22.*
+    - f90nml 1.4.*
+    - jinja2 3.0.*
+    - jsonschema 4.17.*
+    - pyyaml 6.0.*
+    - {{ python }}
+test:
+  requires:
+    - black
+    - coverage
+    - isort
+    - mypy
+    - pylint
+    - pytest
```

Comment:
One reason we've pinned them in the past was because we wanted consistency with supported packages on RDHPCS platforms, i.e., pylint passes on Hera/Jet, but not on GitHub actions because of the version difference.

We may need to go on deep dive (along with some of our EPIC colleagues) to see what makes sense for supporting on all the platforms.